README.md

```diff
@@ -47,6 +47,11 @@ Encode a transaction object into a hex-string. Note that encode filters out fiel
   * If other fields (in the future) must to support X-addresses with tags, this library will need to be updated.
   * When decoding rippled binary, the output will always output classic address + tag, with no X-addresses. X-address support only applies when encoding to binary.
 
+#### Encoding Currency Codes
+  * ripple-binary-codec allows for any 3 character ASCII string to be encoded into a [currency code](https://xrpl.org/currency-formats.html#currency-codes).
+  * ripple-binary-codec will decode currencies into that are valid ISO 4217 codes.
+  * If a currency is not a valid ISO 4217 code, it will return a 160-bit hex-string.
```

Comment:
```suggestion
  * When decoding, if a currency code is does not match the regex, then it is not considered to be an ISO 4217 or pseudo-ISO currency. ripple-binary-codec will return a 160-bit hex-string (40 hex characters). For example, `0000000000000000000000006142430000000000` (`aBC`) decodes as `0000000000000000000000006142430000000000` because it contains a lowercase letter.
```